rearranged files and fixed imports

rymaju opened the pull request:
- Moved page specific files and components into src/pages/route, reusable components stay in /components

rymaju committed:
rearranged files and fixed imports

rymaju committed:
added window scroll to on home

jackblanc commented on src/App.tsx:
```diff
@@ -2,18 +2,17 @@ import React from 'react';
+import Events from './pages/events/Events';
+import People from './pages/people/People';
+import FAQ from './pages/faq/FAQ';
+import NotFound from './pages/notfound/NotFound';
```
thoughts on renaming the files to `index.tsx`? 
That would allow `import Home from './pages/home'`

rymaju replied:
oo interesting, this also exposes a flaw in my directory structure: the ApplyDeveloper/Designer etc pages need to be in their own folder too

rymaju commented:
- Changed css on Board.tsx (now People.tsx) removing complicated media queries and defaulting to mobile styling on all views.
- added scroll to top on home

rymaju commented:
Its probably easier to just check nothing broke when I moved all those files around by checking out the amplify link.

jackblanc commented on src/pages/people/People.tsx:
```diff
@@ -2,10 +2,10 @@ import React from 'react';
 import { makeStyles } from '@material-ui/core/styles';
 import { Container, Grid, Typography } from '@material-ui/core';
-import { ReactComponent as SVG } from '../svg/people.svg';
+import { ReactComponent as SVG } from '../../svg/people.svg';
```
We could colocate SVGs to the /pages/people (for each) because we don't use any twice right?

rymaju replied:
Very smart, I'll do that right now

jackblanc reviewed (commented):
two suggestions

rymaju committed:
reduced grid size

rymaju committed:
moved svgs

jackblanc commented on public/sitemap.xml:
```diff
@@ -13,6 +13,9 @@
   <url>
     <loc>https://c4cneu.com/apply/brand-designer</loc>
+  </url>
+    <url>
```
this looks not formatted
